provider/status_change.json

```diff
@@ -0,0 +1,169 @@
+    },
+    "data": {
+      "$id": "#/properties/data",
+      "type": "object",
+      "title": "The page of data",
+      "required": [],
+      "properties": {
+        "status_changes": {
+          "$id": "#/properties/data/properties/status_changes",
+          "type": "array",
+          "title": "The status_changes Schema",
+          "items": {
+            "$id": "#/properties/data/properties/status_changes/items",
+            "type": "object",
+            "title": "The Items Schema",
+            "required": [
+              "provider_name",
+              "provider_id",
+              "device_id",
+              "vehicle_id",
+              "vehicle_type",
+              "propulsion_type",
+              "event_type",
+              "event_type_reason",
+              "event_time",
+              "event_location"
+            ],
+            "properties": {
+              "provider_name": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/provider_name",
+                "type": "string",
+                "description": "The public-facing name of the Provider",
+                "examples": [
+                  "Provider Name"
+                ],
+                "pattern": "^(.*)$"
+              },
+              "provider_id": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/provider_id",
+                "description": "The UUID for the Provider, unique within MDS",
+                "$ref": "common.json#/definitions/uuid"
+              },
+              "device_id": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/device_id",
+                "description": "A unique device ID in UUID format",
+                "$ref": "common.json#/definitions/uuid"
+              },
+              "vehicle_id": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/vehicle_id",
+                "type": "string",
+                "description": "The Vehicle Identification Number visible on the vehicle itself",
+                "default": "",
+                "examples": [
+                  "ABC123"
+                ],
+                "pattern": "^(.*)$"
+              },
+              "vehicle_type": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/vehicle_type",
+                "type": "string",
+                "description": "The type of vehicle",
+                "enum": [
+                  "bicycle",
+                  "scooter"
+                ]
+              },
+              "propulsion_type": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/propulsion_type",
+                "type": "array",
+                "description": "The type of propulsion; allows multiple values",
+                "items": {
+                  "type": "string",
+                  "enum": [
```

Comment:
Need to add `electric_assist` to this enum, CityOfLosAngeles/mobility-data-specification#48